examples/Catena4430_Sensor/Catena4430_cMeasurementLoop.h

```diff
@@ -382,6 +382,10 @@ class cMeasurementLoop : public McciCatena::cPollableObject
     bool handleSdFirmwareUpdate();
     bool handleSdFirmwareUpdateCardUp();
     bool updateFromSd(const char *sFile, McciCatena::cDownload::DownloadRq_t rq);
+    void handleSdTTNv3Migrate();
+    bool handleSdTTNv3MigrateCardUp(const char *sFile);
```

Comment:
I don't think this function is warranted.

Reply:
have declared the function with void argument, as file name is declared global. and that works